examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>"This will make sure a custom segmentation head is created" (will be -&gt; is)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>